README.md

```diff
@@ -343,6 +346,78 @@ For example, if we named the above XML launch script, `record_all.launch.xml`:
 $ ros2 launch record_all.launch.xml
 ```
 
+You can also invoke the `play` and `record` functionalities as nodes. Nodes `player` and `recorder` are available in `rosbag2_transport` package. The advantage to use this invocation strategy is that the Python layer handling the `ros2 bag` CLI is completely skipped.
```

Comment:
The advantage of skipping python layer looks like a very weak argument.  Because our python layer is a very thin wrapper around CPP API.
it would be worth mentioning that it gives the ability to provide settings for the rosbag2 via node config files and use intra-process communication for better efficiency by avoiding extra data copies on pub-sub transport layer.